charts/github-bot-safe-settings/values.yaml

```diff
@@ -0,0 +1,127 @@
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 5
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+appEnv:
+  enabled: true
+  create: false
+  name: "controller-manager"
+  annotations: {}
+  NODE_EXTRA_CA_CERTS: ""
+
+deploymentConfig:
+  # Set restrictions on repositories
+  restrictedRepos:
+    # You can exclude certain repos from safe-settings processing
+    # If no file is specified, then the following repositories - 'admin', '.github', 'safe-settings', '.*-test' are exempted by default
+    exclude: ['^admin$', '^\.github$', '^safe-settings$', '.*-test']
+    # Alternatively you can only include certain repos
+    include: ['^test$']
+  # No collaborator should be given admin permission
+  configvalidators:
+    - plugin: collaborators
+      error: |
+        `Admin cannot be assigned to collaborators`
+      script: |
+        console.log(`baseConfig ${JSON.stringify(baseconfig)}`)
+        return baseconfig.permission != 'admin'
+  # Validators check whether certain configuration options can be overridden
+  overridevalidators:
+    # Validation of branch security settings to prevent the value of the number of required approvals from being reduced when overriding branch security settings.
+    - plugin: branches
+      error: |
+        `Branch protection required_approving_review_count cannot be overidden to a lower value`
+      script: |
+        console.log(`baseConfig ${JSON.stringify(baseconfig)}`)
+        console.log(`overrideConfig ${JSON.stringify(overrideconfig)}`)
+        if (baseconfig.protection.required_pull_request_reviews.required_approving_review_count && overrideconfig.protection.required_pull_request_reviews.required_approving_review_count ) {
+          return overrideconfig.protection.required_pull_request_reviews.required_approving_review_count >= baseconfig.protection.required_pull_request_reviews.required_approving_review_count
+        }
+        return true
+    # Checking label settings. Does not perform any specific checks on label settings and always returns true
```

Comment:
Shouldn't we add here check for `devops` project when PR is opened, for JIRA task in PR description, for `env` and `Needs Review` labels?

Reply:
no, the secure-settings application only performs the functions of prohibiting the creation of public repositories and creates the necessary labels in the PR
all other functions will be performed by other applications